app/data/models/breach-category.js

```diff
@@ -0,0 +1,57 @@
+module.exports = (sequelize, DataTypes) => {
+  const breachType = sequelize.define('breach_category', {
```

Comment:
Is it a list f breach 'categories' or breach 'reasons'? Picky but I'd say reasons as category would suggest a set of reasons sit within a category

Reply:
Hmmm...realise that 'category/ies' is throughout the code so would be a significant change to rename. Happy for it to be left as breach_category